pkg/chart/chart.go

```diff
@@ -311,7 +313,11 @@ func (c *Chart) InitNetClient(customCA *CustomCA) (HTTPClient, error) {
 		}
```

Comment:
not related to your changes but re-reading this I think we should return the error (as you added) instead of logging a fatal error.

Reply:
Woah, most definitely. I'll do that and the one I actually meant (up on 301 above), in the next PR.